File: programmers/js/new/level1/이상한_문자_만들기.js
function solution(s) {
    var answer = "";
    s.split(" ").forEach((word) => {
        for (let i = 0; i < word.length; ++i) {
            if (i % 2 === 0) {
                answer += word.charAt(i).toUpperCase();
            } else {
                answer += word.charAt(i).toLowerCase();
            }
        }
        answer += " ";
    });
    return answer.slice(0, -1);
    // return answer.substring(0, answer.length-1);
}

File: programmers/python/level1/이상한_문자_만들기.py
def solution(s):
    answer = ''
    for word in s.split(' '):
        for i, c in enumerate(word):
            if c.isalpha():
                if(i % 2 == 0):
                    answer += c.upper()
                else:
                    answer += c.lower()
            else:
                answer += c
        answer += ' '
    return answer[:-1]

print(solution("try1 hello world"))